### PR TITLE
manifest:  enable SELinux booleans

### PR DESCRIPTION
**Pull request by miabbott**
These two commits enable the following SELinux booleans that are needed for OpenShift operation:

  - `container_use_cephfs`
  - `virt_use_samba`

The first boolean in the list is a backport of something that is currently enabled in RHCOS.  It allows for Ceph volumes to be mounted into containers.

The second boolean in the list is a new request via [RHBZ#1754825](https://bugzilla.redhat.com/show_bug.cgi?id=1754825) that would enable privileged containers to mount in Azure files.

### CODE REVIEW

**Review by ashcrow — APPROVED** [review]

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ postprocess:
     #!/usr/bin/env bash
     set -xeuo pipefail
     setsebool -P -N container_use_cephfs on  # RHBZ#1692369
+    setsebool -P -N virt_use_samba on  # RHBZ#1754825
```
Comment (jlebon) [inline fedora-coreos-base.yaml:97]: Do we also want `virt_use_nfs` here?

**Review by jlebon — APPROVED** [review]
LGTM, just one comment!

(Though it seems like the origin package would be a better place for these.)

File: fedora-coreos-base.yaml (hunk 1 above)
Comment (miabbott) [reply fedora-coreos-base.yaml:97]: The default SELinux policy had it already enabled (and `virt_sandbox_use_all_caps` too):

```
[core@coreos ~]$ rpm-ostree status
State: idle
AutomaticUpdates: disabled
Deployments:
* ostree://fedora:fedora/x86_64/coreos/testing-devel
                   Version: 30.20190924.dev.2 (2019-09-24T14:07:33Z)
                    Commit: cd46b5ca68e7c897758a29e7e642d8f1dba19539f1edae1d034ddf1d06b22c11
              GPGSignature: (unsigned)
[core@coreos ~]$ getsebool virt_use_nfs
virt_use_nfs --> on
[core@coreos ~]$ getsebool virt_sandbox_use_all_caps
virt_sandbox_use_all_caps --> on
```

Comment (jlebon) [reply fedora-coreos-base.yaml:97]: Ahh gotcha, thanks for checking. Will leave open for a bit in case anyone has any concerns.

**Comment by miabbott** [comment]
Pushed a small change to include a link to a tracker issue (coreos/fedora-coreos-tracker#284)

**Pull request merged**